XX.md

```diff
@@ -0,0 +1,266 @@
+    ["d", "<collection identifier>"],
+    ["name", "<collection name>"],
+    // Optional tags
+    ["image", "<collection image>"],
+    ["summary", "<collection description>"],
+    ["location", "<location string>"],
+    ["a", "30402:<pubkey>:<product-id>"], // Product references
+    ["shipping", "<shipping-option-id>"] // References to shipping options
+  ]
+}
+```
+
+### Shipping Option (Kind: 30406)
+Dedicated event for shipping details:
+
+```jsonc
+{
+  "kind": 30406,
+  "created_at": <unix timestamp>,
+  "content": "<optional shipping description>",
+  "tags": [
+    ["d", "<shipping identifier>"],
+    ["name", "<shipping method name>"],
+    ["price", "<base_cost>", "<currency>"],
+    // ["location", "<service area>"],
+    // ["duration", "<estimated delivery time>"]
+    // Maybe we can use `L`and `l` tags for ontology, and use proper ISO3166 for shipping zones, also constraints for sizes and weight
+  ]
+}
+```
+
+## Order Communication Flow
+
+- Order processing uses [NIP-17](17.md) encrypted direct messages. Message direction is determined by the `p` tag - when sent from buyer to merchant, `p` contains the merchant's pubkey, and when sent from merchant to buyer, `p` contains the buyer's pubkey. 
+- The payment request message can be initiated in two ways, depending on whether the merchant has a server handling payments
+
+### Message Types
+
+1. Order Creation (buyer → merchant) (subject "order-info")
+```jsonc
+{
+  "kind": 14,
+  "tags": [
+    ["p", "<merchant-pubkey>"],
+    ["subject", "order-info"],
+    ["order", "<order-id>"],
+    ["item", "<product-id>", "<quantity>"],
+    ["item", "<product-id>", "<quantity>"], // Multiple items possible
+    ["shipping", "<shipping-option-id>"],
+    ["address", "<shipping-address>"],
+    ["email", "<customer-email>"],
+    ["phone", "<customer-phone>"]
+  ],
+  "content": "Additional notes: Please gift wrap the items."
+}
+```
+
+2. Payment Request (merchant doesnt have a payment server) (merchant → buyer ) (subject "order-payment")
+```jsonc
+{
+  "kind": 14,
+  "tags": [
+    ["p", "<buyer-pubkey>"],
+    ["subject", "order-payment"],
+    ["order", "<order-id>"],
+    ["amount", "<total-amount>", "<currency>"],
+    ["payment", "lightning", "<bolt11-invoice>"],
+    ["payment", "bitcoin", "<btc-address>"],
+    ["expiry", "<unix-timestamp>"]
+  ],
+  "content": "Payment is due within 24 hours."
+}
+```
+
+3. Payment Request (merchant have a payment server) ( buyer → merchant ) (subject "order-payment")
+```jsonc
+{
+  "kind": 14,
+  "tags": [
+    ["p", "<merchant-pubkey>"],
+    ["subject", "order-payment"],
+    ["order", "<order-id>"],
+    ["amount", "<total-amount>", "<currency>"],
+    ["payment", "lightning", "<bolt11-invoice>"],
+    ["payment", "bitcoin", "<btc-address>"],
+    ["expiry", "<unix-timestamp>"]
+  ],
+  "content": "Payment details provided by merchant's payment server."
+}
+```
+4. Payment Receipt (buyer → merchant) (subject "order-receipt")
```

Comment:
Shouldn't receipts be a kind on its own? This is the only message that is not a "temporary" update. I can forsee users wanting to see a list of all of their purchases. It would be nice to have a separate kind so that there is no need to parse all DMs to find all receipts.